build-tools/packages/version-tools/src/internalVersionScheme.ts

```diff
@@ -20,11 +20,17 @@ const MINIMUM_PUBLIC_MAJOR = semver.major(MINIMUM_PUBLIC_VERSION);
 /** The minimum number of prerelease sections a version should have to be considered a Fluid internal version. */
 const MINIMUM_SEMVER_PRERELEASE_SECTIONS = 4;
 
+/**
+ * The first part of the semver prerelease value is called the "prerelease identifier". For Fluid internal versions,
+ * this is the default prerelease indentifier.
+ */
+export const DEFAULT_PRERELEASE_IDENTIFIER = "internal";
+
 /**
  * The first part of the semver prerelease value is called the "prerelease identifier". For Fluid internal versions, the
- * value must always match this constant.
+ * value must always match one of these values.
  */
-export const REQUIRED_PRERELEASE_IDENTIFIER = "internal";
```

Comment:
(Why isn't GitHub allowing me to comment on lines not changes? CodeFlow let me, but I think those can get overlooked.)
There are remaining cases of `REQUIRED_PRERELEASE_IDENTIFIER` in the bigger comments.
The bigger comments and naming are very focused on `internal` and should be updated at some point, if possible.

Reply:
Part of the challenge is that the name of the version scheme is the "Fluid internal version scheme." The semi-formal definition even says, "Fluid internal version strings _always_ include the string internal in the first position of the pre-release section." My original goal was to try and keep the definition as narrow and inflexible as possible. Eventually had to relax it because of dev releases and now this RC version. Maybe we just need to rename the version scheme to something other than "Fluid internal version scheme." But I'm leery of making it sound _too_ official/functional.